app/views/hyrax/dashboard/sidebar/_repository_content.html.erb

```diff
@@ -0,0 +1,15 @@
+<li class="h5"><%= t('hyrax.admin.sidebar.repository_objects') %></li>
+  <% if can?(:manage, :collection_types) %>
```

Comment:
Could not find a helper, open to feedback on this conditional

Reply:
How about `can?(:manage, Collection)`?  That would make sense to me.

Reply:
I think we'll also want to add this as a feature flipper. See Paul's PR for how to do this: https://github.com/ubiquitypress/hyku_addons/pull/164/files
So the logic would become "if feature enabled and can manage Collection"

Reply:
Sure I'll take a look